spec/commands/doctor_spec.rb

```diff
@@ -58,4 +57,51 @@
        * rack: /usr/local/opt/icu4c/lib/libicui18n.57.1.dylib
     E
   end
+
+  it "exits with an error if home contains files that are not readable/writable" do
```

Comment:
These tests can be wrapped with context blocks.
```
it 'exits with an error if home contains files that are not readable/writable'`
```

can be better written as:

```
context 'when home contains files that are not readable/writable'
    it 'exits with an error'
```

It also means that we can move some of setup for these tests into `before` blocks

(Same applies to the other specs)